src/maven-repair/src/main/java/com/github/tdurieux/repair/maven/plugin/NPEFixMojo.java

```diff
@@ -192,7 +196,7 @@ public void execute() throws MojoExecutionException {
             jsonObject.write(writer);
```

Comment:
try should start before the actual repair attempt

Reply:
Done here.